code/parachain/frame/composable-traits/src/assets.rs

```diff
@@ -34,3 +35,136 @@ pub struct Asset<Balance, ForeignId> {
 	pub foreign_id: Option<ForeignId>,
 	pub existential_deposit: Balance,
 }
+
+/// Struct containing the information used to create an asset
+#[derive(Decode, Encode, Debug, Clone, PartialEq, Eq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct AssetInfo<Balance> {
+	/// Name of the asset.
+	pub name: Vec<u8>,
+	/// Symbol of the asset.
+	pub symbol: Vec<u8>,
+	/// The number of decimals this asset uses to represent one unit.
+	pub decimals: u8,
```

Comment:
We have `Exponent` type alias used elsewhere

Reply:
7d40673a5d8f5d5d268c3b6dc95947da8e02794b